components/formats-bsd/test/loci/formats/utests/SixteenBitLosslessJPEG2000Test.java

```diff
@@ -66,12 +66,21 @@
   private static final Logger LOGGER =
     LoggerFactory.getLogger(SixteenBitLosslessJPEG2000Test.class);
 
+  private int increment = 1024;
+
+  public SixteenBitLosslessJPEG2000Test() {
+  }
```

Comment:
No, as how the no-arg constructor could specify 1024 default. (Fewer state changes = easier reasoning.)